.github/workflows/playwright.yml

```
@@ -105,6 +105,8 @@ jobs:
       - name: Build application
         run: npm run build
       - name: Run Playwright tests (shard 1 of 3)
+        env:
+          OPENAI_TOKEN: ${{ secrets.OPENAI_TOKEN }}
```

actionlint (1.7.10)

property "openai_token" is not defined in object type {actions_runner_debug: string; actions_step_debug: string; github_token: string; run_browser_tests: string}

(expression)